src/utils.js

```diff
@@ -23,7 +23,11 @@ const reg = require("./reg.js");
 
 async function displayError(req, res, details) {
   console.error(details);
-  res.status(500).render('error', details);
+  if (details?.status_to_display && typeof details.status_to_display === "number") {
```

Comment:
```suggestion
  if (typeof details?.status_to_display === "number") {
```

This is an optional thing to streamline it, but I kind of think it is fine as-is to more clearly in plain English explain the two checks (is it there/truthy, and is it a number type?). So, honestly either syntax/style of expressing this being functionally equivalent, go for what style you like.